functions/mod.h

```diff
@@ -0,0 +1,59 @@
+#ifndef FUNCTIONS_MOD_F_H
+#define FUNCTIONS_MOD_F_H
+
+#include "svf.h"
+
+class BladeBase;
+
+template<class F, class MAX>
+class ModBase {
+public:
+  void run(BladeBase* blade) {
+    f_.run(blade);
+    max_.run(blade);
+  }
+  int getInteger(int led) {
+    if (max_.getInteger(led) == 0) {
```

Comment:
That is indeed what I meant, but we shouldn't call max_.getInteger twice.
We should stick the return value in a variable and use that twice instead.
Same goes for max_.calculate below.

Reply:
should've know that, fixed

Reply:
just pinging, the MOD fix in stripes corrected usage